frontend/src/pages/modelRegistry/screens/components/DeployRegisteredModelModal.tsx

```diff
@@ -52,11 +52,10 @@ const DeployRegisteredModelModal: React.FC<DeployRegisteredModelModalProps> = ({
   } = useRegisteredModelDeployInfo(modelVersion);
 
   const onClose = React.useCallback(
-    (submit: boolean) => {
+    async (submit: boolean) => {
```

Comment:
I'm not sure I see why this was made async, I can't find anywhere it's awaited and it's not awaiting anything